webapp/src/shared/components/Icon/Icon.tsx

```diff
@@ -45,8 +45,8 @@ const Icons: { [key in IconVariant]: SVGComponent } = {
 };
 
 export enum IconSize {
-  SMALL = '24px',
-  LARGE = '48px',
+  SMALL = '1.25rem',
```

Comment:
Adjust icon size, in desktop where too big

Reply:
In Firefox, there is this console warning, but it works ok from what I can tell:

```
Unexpected value 1.25rem parsing height attribute.                   react-dom.development.js:798
Unexpected value 1.25rem parsing width attribute.                    react-dom.development.js:798
```

Reply:
I added this so we could have scaling depending on the device which makes sense because we don't want to see icons with the same size in all screens.

Maybe we can just inline css in the Icon component. It's not the best practise but, since we don't have any more styles here it seems to be a good fit. Something like:

```tsx
  <IconTag className={color} style={{width: size, height: size}} />;
```

Reply:
According to [this](https://stackoverflow.com/a/46337386) SO comment, Firefox does not support rem for SVGs.
Maybe we could wrap the Icon component in a div and apply the size with rem to the div instead because the SVGs can scale to the size of their container.

What do you think?